cmd/akash/cmd/root.go

```diff
@@ -190,6 +190,8 @@ func queryCmd() *cobra.Command {
 
 	app.ModuleBasics().AddQueryCommands(cmd)
 	cmd.PersistentFlags().String(flags.FlagChainID, "", "The network chain ID")
+	cmd.PersistentFlags().String(flags.FlagKeyringBackend, flags.DefaultKeyringBackend,
+		"Select keyring's backend (os|file|kwallet|pass|test)")
```

Comment:
Why do we need either of these for query commands?

Reply:
Good catch, I must have misattributed an error to the flag, after rerunning some of the query commands I made, it's not the keyring. Removed, and updated commit message.